### PR TITLE
Fix Netflow reconciliation

### PR DESCRIPTION
**Pull request by tanyatukade**
The hash tree comparison fails due to integer types being converted to string values when creating the APIC-side hash tree. These string values result in different hashes in the hash tree for those resources, and therefore AID never reaches sync. This patch converts the integer values used by Netflow Rs object to strings.

Signed-off-by: Tanya Tukade tanyatukade.123@gmail.com

### CODE REVIEW

**Comment by tanyatukade** [comment]
- The tox pep8 test fails because the db migration script uses previous PR's migration down revision as the revision ID.
- I will wait until the previous PR is merged into master and rerun the test.

**Review by tbachman — APPROVED** [review]
Looks good to me. We can rebase this with the new DB HEAD once the ERSPAN patch merges.

**Comment by tanyatukade** [comment]
Rebased with master

**Pull request merged**
